packages/schemas/src/date/helper.ts

```diff
@@ -56,100 +147,124 @@ const injectStyles = (css: string) => {
     }
     return new Date(`2021-01-01T${strDate}`);
   }
+
   return new Date(strDate);
 };
 
-export const getPlugin = ({
-  type,
-  defaultFormat,
-  icon,
-  formatsByLang,
-}: {
-  type: 'date' | 'time' | 'dateTime';
-  defaultFormat: string;
-  icon: string;
-  formatsByLang: Record<Lang, string[]>;
-}) => {
+const getFormat = (type: PickerType, locale: Locale): string => {
+  switch (type) {
+    case 'date': {
+      return locale.adLocale.dateFormat;
+    }
+    case 'time': {
+      return 'HH:mm';
+    }
+    case 'dateTime': {
+      return `${locale.adLocale.dateFormat} ${locale.adLocale.timeFormat}`;
+    }
+  }
+};
+
+const getFmtValue = (
+  value: string,
+  type: PickerType,
+  schema: DateSchema,
+  locale: Locale
+): string => {
+  return value
+    ? format(strDateToDate(value, type), schema.format, {
+        locale: locale.formatLocale,
+      })
+    : '';
+};
+
+const getFmtContent = (date: Date[], type: PickerType) => {
+  const fmt = (() => {
+    switch (type) {
+      case 'date': {
+        return 'yyyy/MM/dd';
+      }
+      case 'time': {
+        return 'HH:mm';
+      }
+      case 'dateTime': {
+        return 'yyyy/MM/dd HH:mm';
+      }
+    }
+  })();
+  return date[0] ? format(date[0], fmt) : '';
+};
+
+export const getPlugin = ({ type, icon }: { type: PickerType; icon: string }) => {
+  const defaultLocale = 'en';
+  const defaultFormat = getFormat(type, getAirDatepickerLocale(defaultLocale));
+
   const plugin: Plugin<DateSchema> = {
-    ui: (arg) => {
-      const { schema, value, onChange, rootElement, mode, options, i18n } = arg;
-      injectStyles(airDatepickerCss);
+    ui: async (arg) => {
+      const { schema, value, onChange, rootElement, mode, options } = arg;
 
-      const beforeRemoveEvent = new Event('beforeRemove');
-      rootElement.dispatchEvent(beforeRemoveEvent);
+      const locale = getAirDatepickerLocale(schema.locale || options.lang || defaultLocale);
 
-      const textStyle: CSS.Properties = {
-        fontFamily: schema.fontName ? `'${schema.fontName}'` : 'inherit',
-        color: schema.fontColor ?? DEFAULT_FONT_COLOR,
-        fontSize: `${schema.fontSize ?? DEFAULT_FONT_SIZE}pt`,
-        letterSpacing: `${schema.characterSpacing ?? DEFAULT_CHARACTER_SPACING}pt`,
-        textAlign: schema.alignment ?? DEFAULT_ALIGNMENT,
-        backgroundColor: getBackgroundColor(value, schema),
-
-        margin: '0',
-        padding: '0',
-        border: 'none',
-        outline: 'none',
+      const textElement = document.createElement('div');
+      const textElementStyle: CSS.Properties = {
         width: `${schema.width}mm`,
         height: `${schema.height}mm`,
         display: 'flex',
         flexDirection: 'column',
         justifyContent: mapVerticalAlignToFlex(VERTICAL_ALIGN_MIDDLE),
-        position: 'relative',
       };
+      Object.assign(textElement.style, textElementStyle);
 
-      const input = document.createElement('input');
+      await text.ui(
```

Comment:
By rendering using `text.ui`, you can intentionally display Tofu in the browser even when the font is unavailable.